airbyte-integrations/connectors/source-hubspot/unit_tests/utils.py

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+from typing import Any, MutableMapping
+
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources.streams import Stream
+
+
+def read_incremental(stream_instance: Stream, stream_state: MutableMapping[str, Any]):
```

Comment:
I see this in various PRs, we should make a ticket to add it to the CDK as a helper utility

Reply:
Created https://github.com/airbytehq/airbyte/issues/11324